src/index.js

```diff
@@ -6,6 +6,13 @@ import matter from 'gray-matter'
 import * as esbuild from 'esbuild'
 import {NodeResolvePlugin} from '@esbuild-plugins/node-resolve'
 import {globalExternals} from '@fal-works/esbuild-plugin-global-externals'
+import dirnameMessedUp from './dirname-messed-up.cjs'
+
+if (dirnameMessedUp) {
```

Comment:
Should we add a check here to see if `process.env.ESBUILD_BINARY_PATH` is set and silence this warning if it is?

Reply:
Oh right! Of course. Thanks 👍